mbed_devices/mbed_devices.py

```diff
@@ -25,13 +28,19 @@ def _build_device(candidate: CandidateDevice) -> Device:
     )
 
 
+class NoTargetForCandidate(Exception):
+    """Raised when target cannot be determined for a candidate."""
+
+    pass
```

Comment:
Don't need `pass`

Reply:
Damn it, I'll never learn.

Reply:
https://github.com/ARMmbed/mbed-devices/pull/40/commits/0ca056e5875395b624463aca3acb354859fa5485